tpu-client-next/tests/connection_workers_scheduler_test.rs

```diff
@@ -433,10 +436,10 @@ async fn test_staked_connection() {
 
     // Wait for the exchange to finish.
     tx_sender_shutdown.await;
-    let localhost_stats = join_scheduler(scheduler_handle).await;
+    let localhost_stats = join_scheduler(scheduler_handle).await.to_non_atomic();
```

Comment:
Would it make sense to put the `.to_non_atomic()` call inside the `join_scheduler()`?
Or we do not want it there for non-test case?

Reply:
hm, maybe it makes sense since after the execution we don't need these counters to be atomic anylonger